__main__.py

```diff
@@ -3,81 +3,149 @@
 import datetime as dt
 import json
 import sys
+from enum import Enum
 from pathlib import Path
-from typing import NoReturn
+from typing import TYPE_CHECKING, Any, ClassVar, NewType, NoReturn, TypedDict, cast
 
 import requests
 from dahlia import dahlia, dprint
+from requests.exceptions import HTTPError
+
+if TYPE_CHECKING:
+    from typing_extensions import Self
+
+
+__version__ = "0.1.0"
+
+
+class Status(str, Enum):
+    ACCEPTED = "Accepted"
+    ACTIVE = "Active"
+    DEFERRED = "Deferred"
+    DRAFT = "Draft"
+    FINAL = "Final"
+    PROVISIONAL = "Provisional"
+    REJECTED = "Rejected"
+    REPLACED = "Replaced"
+    WITHDRAWN = "Withdrawn"
+    SUPERSEDED = "Superseded"
+
 
 COLOR_CODES = {
-    "Accepted": "&2",
-    "Active": "&a",
-    "Deferred": "&6",
-    "Draft": "&8",
-    "Final": "&1",
-    "Provisional": "&5",
-    "Rejected": "&4",
-    "Replaced": "&e",
-    "Withdrawn": "&d",
+    Status.ACCEPTED: "&2",
+    Status.ACTIVE: "&a",
+    Status.DEFERRED: "&6",
+    Status.DRAFT: "&8",
+    Status.FINAL: "&1",
+    Status.PROVISIONAL: "&5",
+    Status.REJECTED: "&4",
+    Status.REPLACED: "&e",
+    Status.WITHDRAWN: "&d",
+    Status.SUPERSEDED: "",  # FIXME Pick color
 }
-LATEST = Path("pep-tracker/latest.json")
+
+LATEST = Path(__file__).parent.resolve() / "latest.json"
 URL = "https://peps.python.org/api/peps.json"
+PEP = NewType("PEP", str)
```

Comment:
This is purely for type-hinting and type-checking. The type behaves exactly like a `str`.